Jenkinsfile

```diff
@@ -57,8 +57,12 @@ node(label: 'jenkins-slave') {
       
       parallel (
        'dev': {
-         stdout = sh returnStdout: true, script: 'make s3copybranch PROJECT='+ project +   ' DEPLOY_TARGET=dev DEPLOY_GIT_BRANCH=' + deployGitBranch
-         echo stdout
+         if (project == 'mf-geoadmin3') {
+           stdout = sh returnStdout: true, script: 'make s3copybranch PROJECT='+ project +   ' DEPLOY_TARGET=dev DEPLOY_GIT_BRANCH=' + deployGitBranch
+           echo stdout
+         } else {
```

Comment:
I would prefer to specify what `else` is with a `else if`, so that it auto-documents itself